cms/CONTENTFUL.md

```diff
@@ -0,0 +1,49 @@
+# Contentful
+
+- [EY Recovery Service](https://app.contentful.com/spaces/dvmeh832nmjc)
+- [API Keys](https://app.contentful.com/spaces/dvmeh832nmjc/api/keys/) can be granted access to specific environments or aliases.
+- [Management Tokens](https://app.contentful.com/spaces/dvmeh832nmjc/api/cma_tokens) offer [per-developer](https://app.contentful.com/account/profile/cma_tokens) write access.
+- [Environments](https://app.contentful.com/spaces/dvmeh832nmjc/settings/environments) are accessed via aliases called `master` (production), `staging` and `test`. Demo content is used in `RAILS_ENV=development|test`.
+- [User roles](https://app.contentful.com/spaces/dvmeh832nmjc/settings/users) Content team (editor), Dev team (admin).
+
+
+## Tasks
+
+Contentful tasks are namespaced under `eyfs:cms`, list them using `rake --tasks eyfs:cms`.
+
+- `rake eyfs:cms:migrate`  # Run migration files in `./cms/migrate`
+- `rake eyfs:cms:seed`     # Populate Contentful entries using YAML
```

Comment:
When I ran this locally I got the following:
```
sh: contentful: not found
sh: contentful: not found
sh: contentful: not found
sh: contentful: not found
```
Performed `bin/docker-prune` and ran again - this time:

```
![CleanShot 2023-02-03 at 13 13 05@2x](https://user-images.githubusercontent.com/6605/216612074-46ceb29b-43a6-45fc-838d-82ecc7aa862d.png)

Reply:
Deletion should be done only if it is found to exist.